templates/jlink/jlinked-app.yaml

```diff
@@ -44,9 +47,9 @@ objects:
 - apiVersion: build.openshift.io/v1
   kind: BuildConfig
   metadata: 
-    name: jlink-builder-jdk-${JDK_VERSION}
+    name: ${APPNAME}-jlink-builder-jdk-${JDK_VERSION}
     labels:
-      app: jlink-builder-jdk-${JDK_VERSION}
+      app: ${APPNAME}-jlink-builder-jdk-${JDK_VERSION}
```

Comment:
I wonder if we could just use `label.app: ${APPNAME}` throughout here… once my CRC instance rebuilds I'll see what it looks like

Reply:
let's not block on that